Entity/Repositories/MailgunEventRepository.php

```diff
@@ -250,4 +250,27 @@ public function getEventsBySeverity($severity = MailgunEvent::SEVERITY_INFO, $ma
 
         return $results;
     }
+
+    /**
+     * Get last MailgunEvent
+     *
+     * @return MailgunEvent
+     */
+    public function getLastEvent()
```

Comment:
From what I can see, this function is never called by anyone. I'll remove it, unless you object.

Reply:
sure, no problem. thanks